scripts/start-dev-with-auto-restart.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Auto-restart webpack-dev-server on memory crashes
+# This script monitors for heap out of memory errors and automatically restarts
+# Usage: ./start-dev-with-auto-restart.sh [webpack-dev-server arguments]
+
+WDS_ARGS=("$@")
```

Comment:
NAB but I'd prefer not to abbreviate here:

```suggestion
WEBPACK_DEV_SERVER_ARGS=("$@")
```